confight.py

```diff
@@ -214,12 +214,13 @@ def load_ini(stream):
 
 # Optional dependency yaml
 try:
-    import ruamel.yaml as yaml
+    from ruamel.yaml import YAML
 except ImportError:
     pass
 else:
     def load_yaml(stream):
-        return yaml.load(stream, Loader=yaml.RoundTripLoader)
+        yaml = YAML(typ="rt")
+        return yaml.load(stream)
```

Comment:
Tested using global dependencies for the debian package global installation:

- Ubuntu 20.04.6 LTS focal `python3-ruamel.yaml_0.15.89-3build1_amd64.deb`
- Ubuntu 18.04.6 LTS bionic `python3-ruamel.yaml_0.15.34-1_amd64.deb`

```
>>> from ruamel.yaml import YAML
>>> yaml = YAML(typ="rt")
>>> yaml.load('a: 2')
CommentedMap([('a', 2)])
```

OK